src/ts/client/grapesjs/index.ts

```diff
@@ -437,13 +438,145 @@ export async function initEditor(config: EditorConfig) {
             : notificationButton?.classList.remove('project-bar__dirty')
         }
       )
-
       // GrapesJs editor is ready
       resolve(editor)
     })
   })
 }
 
+// Will be added later in a pluggin but for the test I put it here
+function addThemeSelector(editor) {
+  setTimeout(() => {
+    // Add a selector besides the deviceManager
+    const viewsBar = document.querySelector('.gjs-pn-panel.gjs-pn-devices-c')
```

Comment:
I believe there is a grapesjs api to get a reference to the option panel, can you check if it can be used here?